### DISCUSSION TITLE
lotus/extern/filecoin-ffi/generated/../filcrypto.pc': No such file or directory

### DISCUSSION THREAD

**Discussion by 16871560** [filecoin-project/lotus#7849] [Build and Sync] [answered]
I am  a new for filelcoin ,I clone the git and cd lotus/cmd/lotus-miner and try to go build it , it show me like this 'lotus/extern/filecoin-ffi/generated/../filcrypto.pc': No such file or directory' ,  does anybody can help, many thx

**Comment by stuberman** [comment, marked as answer]
Here is what I use for build/rebuild

> sudo rm -r lotus
git clone https://github.com/filecoin-project/lotus.git
cd lotus/
git checkout v1.13.1 && git switch -c v1.13.1 && git submodule update --init --recursive
RUSTFLAGS="-C target-cpu=native -g" FFI_BUILD_FROM_SOURCE=1 make clean all lotus-bench lotus-shed
sudo make install
cd ..


(Then run lotus daemon, etc)

**Reply by 16871560** [reply]
thx, I will try it. in fact, I wanna clone the codes to read & learn.

**Comment by LexLuthr** [comment]
I am archiving this discussion as there has been no new input for more than 7 days.
